compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -477,6 +477,10 @@ static void PrintMethodFields(
           "private static volatile $MethodDescriptor$<$input_type$,\n"
           "    $output_type$> $method_new_field_name$;\n"
           "\n"
+          "@$RpcMethod$(\n"
```

Comment:
I think we need to include the method name as well, because the method name → Java method name conversion is lossy. We should either include the "full method name" or the "full service name"+"method name".

Reply:
Done.